Community-Supported/hyper-jupyter-kernel/README.md

```diff
@@ -21,17 +21,19 @@ With the `hyper_kernel` you can integrate HyperAPI's SQL interface directly into
 
 ![Screenshot of jupyter console](media/jupyterlab.png)
 
-First, install jupyter-console and the kernel provided in this repository using
+First, install JupyterLab and the kernel provided in this repository using
 
 ```
-python -m pip install jupyter-console tableauhyperapi git+https://github.com/tableau/hyper-api-samples#subdirectory=Community-Supported/hyper-jupyter-kernel
+python -m pip install jupyterlab tableauhyperapi git+https://github.com/tableau/hyper-api-samples#subdirectory=Community-Supported/hyper-jupyter-kernel
 python -m hyper_kernel.install
 ```
 
+(Depending on your operating system, you might need to replace `python` by `python3`)
+
 Next launch JupyterLab using
 
 ```
-python -m jupyter lab
+python -m jupyter lab --core-mode
```

Comment:
`--core-mode` disables JavaScript from the extensions, right? It shouldn't be necessary to specify it. It might be that it helps on some systems, but I don't see how it's related to this kernel.

Reply:
You need to use `--core-mode` if you don't have nodejs installed.
Without core-mode, jupyterlab will otherwise try to rebuild the JavaScript, and fail to do so if node is missing.

I don't want to teach people how to install node, but just want them to get started with JupyterLab quickly.
Using `--core-mode` allows to bypass the whole "JavaScript-building" issues